arithmetization/src/test/java/net/consensys/linea/zktracer/module/ext/TestDuplicatedOperations.java

```diff
@@ -1,50 +0,0 @@
-/*
- * Copyright Consensys Software Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
- * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
- * specific language governing permissions and limitations under the License.
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package net.consensys.linea.zktracer.module.ext;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-import net.consensys.linea.testing.BytecodeCompiler;
-import net.consensys.linea.testing.BytecodeRunner;
-import net.consensys.linea.zktracer.opcode.OpCode;
-import org.apache.tuweni.bytes.Bytes;
-import org.junit.jupiter.api.Test;
-
-public class TestDuplicatedOperations {
```

Comment:
Why deleting this test ? it looks good for me, I think we should have even more like this one.

Reply:
This test is a duplicate of the `stackContainer` tests, it just verifies that the duplicates are killed, which is the purpose of the tests I added, but are more general.